src/release-build-tools.js

```diff
@@ -351,17 +297,13 @@ module.exports = {
       await prepPackageForRelease(instruction, package, release, workingCopyPath);
     }
 
-    if (instruction.magentoCommunityEditionMetapackage) {
-      // nothing to do - the product-community-edition metapackage composer.json is built from a template
-    }
-
-    if (instruction.magentoCommunityEditionProject) {
-      // update the base composer.json for releasing (doesn't happen for the base package because that uses a composer.json template)
-      const metapackage = new packageDefinition({
-        'label': 'Mage-OS Community Edition Project Metapackage',
-        'dir': ''
+    const communityEdition = (instruction.extraMetapackages || []).find(package => package.name === 'project-community-edition');
+    if (communityEdition) {
+      const package = new packageDefinition({
+        label: communityEdition.description || communityEdition.name,
+        dir: '', // metapackages typically at repo root
```

Comment:
This comment isn't correct - most meta-packages are in a `_metapackage` subdirectory, like https://github.com/mage-os/mageos-security-package/tree/main/_metapackage or https://github.com/mage-os/mageos-inventory/tree/main/_metapackage

I thin only the community edition meta-packages would be in the repo root.

Reply:
Corrected, thanks